rpm/update-config-files.sh

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env bash
+# This script delivers current documentation/configs and assures it has the intended
+# settings for a particular branch/release.
+
+set -exo pipefail
+
+ensure() {
+  if grep ^$2[[:blank:]].*= $1 > /dev/null
+  then
+    sed -i "s;^$2[[:blank:]]=.*;$2 = $3;" $1
+  else
+    if grep ^\#.*$2[[:blank:]].*= $1 > /dev/null
+    then
+      sed -i "/^#.*$2[[:blank:]].*=/a \
+$2 = $3" $1
+    else
+      echo "$2 = $3" >> $1
+    fi
+  fi
+}
+
+# Common options enabled across all fedora, centos, rhel
+# TBD: Can these be enabled by default upstream?
+ensure registries.conf              short-name-mode     \"enforcing\"
+
+ensure storage.conf                 driver              \"overlay\"
+ensure storage.conf                 mountopt            \"nodev,metacopy=on\"
+
+ensure pkg/config/containers.conf   runtime             \"crun\"
+ensure pkg/config/containers.conf   log_driver          \"journald\"
+ensure pkg/config/containers.conf   compression_format  \"zstd:chunked\"         
+
+
+# Enable seccomp support keyctl and socketcall
+[ `grep \"keyctl\", pkg/seccomp/seccomp.json | wc -l` == 0 ] && sed -i '/\"kill\",/i \
+				"keyctl",' pkg/seccomp/seccomp.json
+[ `grep \"socket\", pkg/seccomp/seccomp.json | wc -l` == 0 ] && sed -i '/\"socketcall\",/i \
+				"socket",' pkg/seccomp/seccomp.json
+
+FEDORA=$(rpm --eval %{?fedora})
+RHEL=$(rpm --eval %{?rhel})
+COPR=$(rpm --eval %{?copr_username})
+
+# Set search registries
+if $FEDORA; then
```

Comment:
This does not work: in non-fedora, you'd get `if ` (if blank), syntax error, barf. See diff.